source/javascripts/components/ProductTour/ProductTooltip.tsx

```diff
@@ -36,19 +38,25 @@ export const ProductTooltip = ({
 		onClose();
 	};
 
+	const onStartAgain = (): void => {
+		onButtonClick("start again");
+		onRestart();
+	};
+
 	return (
 		<Box
-			left={rect?.x}
-			top={(rect?.y ?? 0) + (rect?.height ?? 0) + 20}
+			left={tip.position === "right" ? rect!.x + rect!.width + 37: rect!.width/2 - 210}
+			top={tip.position === "right" ? rect!.bottom/2  - 22: (rect?.y ?? 0) + (rect?.height ?? 0) + 40}
 			className="product-tooltip"
+			position="relative"
 		>
 			<Box display="flex" flexGrow="1">
-				<div className="arrow-up" style={{ left: rect!.width / 2 - 5 }} />
+				<div className={`arrow-${tip.position === "right" ? "left" : "up" }`} />
```

Comment:
Now that we have a proper bitkit version, could we switch to the `Popover` component to fix a lot of manual stuff we do here? Render something for the `PopoverTrigger` and we could just delete most of the tooltip positioning related stuff. 

I would do it in a follow up PR if it's not trivial to do, to not interfere with this development.

Reply:
This has been updated. Please see my latest changes